containers/base/Dockerfile

```diff
@@ -123,20 +124,6 @@ RUN echo "deb-src http://ftp.us.debian.org/debian testing main" >> /etc/apt/sour
     rm -rf /usr/share/i18n/locales/* && \
     cd /
 
-# Install ProtoBuf, TensorFlow
```

Comment:
Seems incorrect for pydatalab to bring in TensorFlow or Dataflow. We don't want all users of pydatalab depending on those. The specific features in pydatalab should handle missing dependencies with appropriate error, and we should continue to install these explicitly.